keras/callbacks.py

```diff
@@ -715,6 +715,12 @@ class TensorBoard(Callback):
             `embeddings_layer_names`. Numpy array (if the model has a single
             input) or list of Numpy arrays (if the model has multiple inputs).
             Learn [more about embeddings](https://www.tensorflow.org/programmers_guide/embedding)
+        write_step: `'batch'` or `'epoch'` or integer. When using `'batch'`, writes
```

Comment:
There's already a `embeddings_freq` argument, and this new argument `write_step` is not consistent with it. Maybe `write_freq` or `update_freq` would be better. Thoughts?

Reply:
Naming things is as hard as ever.
I like `update_freq` since we "abstract" the writing to the file in this case. The user doesn't really care (take this with a grain of salt) what happens to the file. He/she wants to see the plot being updated. That's what matters.